### ISSUE TITLE
Chant Search Page - Contains/Starts-with dropdown should be smaller

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#824]
From an email from Debra, where she was forwarding comments she'd received on to us developers:

> > http://206.12.88.113/chant-search/  - Maybe it's just me, but I keep trying to type in the search word in the drop-down box ("Starts with" or "Contains") instead of in the "Keyword" box. I think it might be because they are the same size. Is there something one can easily do about that?
>
> Me too! Now that you say it, I see what you mean. I clicked in the "Contains" box as well, and then realized what I had done. I'm sure we could train ourselves not to do this, but the "Contains" / "Starts with" box doesn't likely have to be that large. OR, how about we switch the order and have the box you type in come first? With "Contains/Start with" beside it to the right? Or is that strange and out of order the way you would think about it?

We should make the contains/starts-with dropdown smaller.

**Comment by jacobdgm** [comment]
This is fixed! https://cantusdatabase.org/chant-search/
